ipaclient/remote_plugins/schema.py

```diff
@@ -64,6 +64,13 @@
 }
 
 
+class BytesEncoder(json.JSONEncoder):
+    def default(self, obj):
```

Comment:
Instead of using a subclass, ``json.dumps()`` also takes a ``default`` function kwarg. The function takes one argument and should raise TypeError in case of an unsupported type.